packages/task/src/node/task-server.ts

```diff
@@ -67,6 +67,10 @@ export class TaskServerImpl implements TaskServer {
         return taskInfo;
     }
 
+    async getRegisteredTaskTypes(): Promise<string[]> {
+        return Promise.resolve(this.runnerRegistry.getRunnerTypes());
```

Comment:
It's the async function, so its return value will be automatically wrapped in a resolved promise
`return this.runnerRegistry.getRunnerTypes();`